test/cases/create_unlogged_tables_test.rb

```diff
@@ -53,7 +53,7 @@ def test_gracefully_handles_temporary_tables
         @connection.create_table(TABLE_NAME, temporary: true) do |t|
         end
 
-        assert_equal @connection.execute(LOGGED_QUERY).first[LOGGED_FIELD], LOGGED
+        assert_equal @connection.execute(LOGGED_QUERY).first[LOGGED_FIELD], TEMPORARY
```

Comment:
it seems like we can just remove this `test_gracefully_handles_temporary_tables` method entirely, and go back to using the upstream test.

I think we have to keep `test_unlogged_in_test_environment_when_unlogged_setting_enabled`, but can you adjust the comment on that one to point at https://github.com/cockroachdb/cockroach/pull/56827 and mention that CockroachDB ignores the `UNLOGGED` specifier

Reply:
This is actually failing once removed since we need the connection to set `experimental_enable_temp_tables`. I could potentially set that earlier in the testing flow (or in the adapter), but it would probably be easier to leave it and track this setting somewhere.

Reply:
Ah ok, sorry for the distraction then! I thought we had done something to enable temp tables everywhere. (It can be done globally by using `SET CLUSTER SETTING sql.defaults.experimental_temp_tables_enabled = true`)

Reply:
Hmm, I tried setting that in the teamcity-test build with 

```
cockroach sql --insecure -e "SET CLUSTER SETTING sql.defaults.experimental_temp_tables_enabled = true"
```

but it raised an error

```
ERROR: unknown cluster setting 'sql.defaults.experimental_temp_tables_enabled
```

I also couldn't find it in the result from `SHOW CLUSTER SETTINGS`.

Reply:
I just reverted it for now, but once we figure out the setting I can remove that test.

Reply:
my bad.. i shouldn't have tried saying the name from memory.

it's `SET CLUSTER SETTING sql.defaults.experimental_temporary_tables.enabled = 'true';`

the next release of cockroachdb will include it in `SHOW CLUSTER SETTINGS`

Reply:
Ok cool that worked thanks! I added that to the teamcity-test setup and removed the `test_gracefully_handles_temporary_tables` test.